clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -16,10 +16,6 @@
  */
 package org.apache.kafka.common.utils;
 
-import java.nio.BufferUnderflowException;
```

Comment:
@manijndl7 Please revert any unnecessary changes in imports. The imports removed are added again at the bottom. Please add or remove only the diff.